locator/token_metadata.hh

```diff
@@ -116,7 +116,7 @@ public:
     const std::unordered_map<token, inet_address>& get_token_to_endpoint() const;
     const std::unordered_set<inet_address>& get_leaving_endpoints() const;
     const std::unordered_map<token, inet_address>& get_bootstrap_tokens() const;
-    void update_topology(inet_address ep, endpoint_dc_rack dr, topology::pending pend = topology::pending::no);
+    void update_topology(inet_address ep, endpoint_dc_rack dr);
```

Comment:
There shouldn't be, but I'll double check.

Reply:
I don't see any need for adjustment.
As I said in https://github.com/scylladb/scylladb/pull/12294#issue-1493732924 and  https://github.com/scylladb/scylladb/pull/12294/files#r1048116078, now that is_member relies on token_metadata rather than on topology, there is no need to keep endpoints in topology in pending state.